README.md

```diff
@@ -220,9 +220,17 @@ Work on ``pandas`` started at AQR (a quantitative hedge fund) in 2008 and
 has been under active development since then.
 
 ## Discussion and Development
-Since pandas development is related to a number of other scientific
-Python projects, questions are welcome on the scipy-user mailing
-list. Specialized discussions or design issues should take place on
-the PyData mailing list / Google group:
+General questions and discussions can take place on the [pydata mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata).
```

Comment:
@jorisvandenbossche 
Just saw your last comment after I submitted the latest change.
I'll move it to Getting Help.